### DISCUSSION TITLE
How did the `recommendations` variable turn from a set to integer?

### DISCUSSION THREAD

**Discussion by saaalik** [Praful932/Kitabe#113] [Q&A]
In the page `views.py` when importing using `combine_ids` from helper.py, `already_rated_books` variable is passed as an input which is a set [image1](https://ibb.co/VJv1Q6C)
but in the function defination that input is used as an integer [image2](https://ibb.co/n0nxGvp)
output of type of `recommendation` variable when combine_ids is executed [image3](https://ibb.co/nbTcJB5)
output of `already_rated_books` [image4](https://ibb.co/9pzB735)
@Praful932 please look into it

**Comment by Praful932** [comment]
@saaalik That is a default parameter 😉, Try printing `already_rated` variable
